### PR TITLE
Add support for extra large dropzone

### PR DESCRIPTION
**Pull request by vladucu**

### CODE REVIEW

File: tests/integration/components/polaris-drop-zone-test.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render, triggerEvent, settled } from '@ember/test-helpers';
+import { pauseTest, render, triggerEvent, settled } from '@ember/test-helpers';
```
Comment (tomnez) [inline tests/integration/components/polaris-drop-zone-test.js]: Left a `pauseTest` in here

Comment (vladucu) [reply tests/integration/components/polaris-drop-zone-test.js]: oh, good catch

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,10 +2,11 @@
 
 ### Unreleased
 - [#144](https://github.com/smile-io/ember-polaris/pull/144) [UPDATE] Shopify Polaris `v2.2.0`
-- [#146](https://github.com/smile-io/ember-polaris/pull/146) Add `new` status to the Badge component
+- [#146](https://github.com/smile-io/ember-polaris/pull/146) [UPDATE] Add `new` status to the Badge component
 - [#148](https://github.com/smile-io/ember-polaris/pull/148) [FEATURE] Add `polaris-label` and `polaris-labelled` components.
 - [#151](https://github.com/smile-io/ember-polaris/pull/151) [FIX] Add missing `hidden` class to `polaris-label`.
-- [#152](https://github.com/smile-io/ember-polaris/pull/152) Accept card title as a string or a component
+- [#152](https://github.com/smile-io/ember-polaris/pull/152) [UPDATE] Accept card title as a string or a component
```
Comment (andrewpye) [inline CHANGELOG.md]: Is this an update or an enhancement? 🤔 

🤷‍♂️

File: addon/components/polaris-drop-zone.js [hunk 3]
```diff
@@ -404,13 +404,15 @@ export default Component.extend(ContextBoundEventListenersMixin, {
   adjustSize() {
     throttle(this, function() {
       let node = this.get('node');
-      let size = 'large';
+      let size = 'extraLarge';
```
Comment (andrewpye) [inline addon/components/polaris-drop-zone.js]: Maybe worth pulling this value into a `defaultSize` constant

File: CHANGELOG.md (hunk 2 above)
Comment (vladucu) [reply CHANGELOG.md]: not sure if it's defined anywhere what's an update or enhancement

File: addon/components/polaris-drop-zone.js (hunk 3 above)
Comment (vladucu) [reply addon/components/polaris-drop-zone.js]: yeah, can do

File: CHANGELOG.md (hunk 2 above)
Comment (andrewpye) [reply CHANGELOG.md]: No, not really. To me an enhancement is maybe every so slightly more specifically about improving existing functionality while an update is more general... with that in mind, it all depends on the wording used in the PR summery as to whether it's an enhancement or an update. Doesn't really matter either way 😛

**Pull request merged**